lualibs/go/ngx/loader.go

```diff
@@ -19,6 +20,24 @@ func LoadNgx(l *lua.LState) {
 	mt := l.NewTable()
 	l.SetFuncs(mt, map[string]lua.LGFunction{"__tostring": NullString})
 	l.SetField(ngx, "null", d)
+
+	// fetch gluare.find and gluare.match functions
+	// see https://github.com/yuin/gluare/blob/master/gluare.go
+	ldr := gluare.Loader(l)
+	ltbl := l.ToTable(ldr)
+	findFn := l.GetField(ltbl, "find")
+	matchFn := l.GetField(ltbl, "match")
```

Comment:
These values could be null, we should handle that case and panic. Granted it shouldn't happen, but let's handle appropriately.